src/tx-builder/services/DydxStateChecker.ts

```diff
@@ -0,0 +1,29 @@
+import { DydxGovernor, DydxGovernor__factory } from '../../../types';
+import { Configuration } from '../types';
+
+export class DydxStateChecker {
+  readonly config: Configuration;
+
+  constructor(config: Configuration) {
+    this.config = config;
+  }
+
+  public async getGovernanceVoters(
```

Comment:
nit: Mind using the [`DydxGovernance` service](https://github.com/dydxfoundation/governance-contracts/blob/master/src/tx-builder/services/DydxGovernance.ts) instead of making a new one?